groupease-server/src/main/java/io/github/groupease/model/ChannelInvitation.java

```diff
@@ -0,0 +1,25 @@
+package io.github.groupease.model;
+import org.hibernate.annotations.UpdateTimestamp;
+import javax.persistence.*;
+import java.time.Instant;
+
+@Entity
+@Table(name="ChannelInvitation")
+public class ChannelInvitation {
+    @Id
+    private Long id;
+    private long channelId;
```

Comment:
This should probably be `Long` so a null can be provided by the user to allow an instance missing the property in the door. The service layer should verify it is present before sending to the DAO layer.